### ISSUE TITLE
## Training and Validation Accuracy (without overfitting)

### ISSUE THREAD

**Issue by AhmadSabbirChowdhury** [AhmadSabbirChowdhury/Cats-Vs-Dogs-Classification-Kaggle-Problem-Set-#3]
![accurate data](https://user-images.githubusercontent.com/45898995/82494273-266f5f00-9b0b-11ea-8568-62483d0356f3.png)
